codecov_example/README.md

```diff
@@ -0,0 +1,317 @@
+codecov_example so the command would be...
+```sh
+mix new codecov_example
+* creating README.md
+* creating .formatter.exs
+* creating .gitignore
+* creating mix.exs
+* creating config
+* creating config/config.exs
+* creating lib
+* creating lib/codecov_example.ex
+* creating test
+* creating test/test_helper.exs
+* creating test/codecov_example_test.exs
+
+Your Mix project was created successfully.
+You can use "mix" to compile it, test it, and more:
+
+    cd codecov_example
+    mix test
+
+Run "mix help" for more commands.
+```
+
+Now let's follow the on screen instructions and `cd` into the directory.
+
+Something I personally think is pretty cool is that this app will already have
+a function and a tests for that function. We can run the tests with `mix test`
+(as our terminal said above)...
+
+```sh
+mix test
+..
+
+Finished in 0.03 seconds
+1 doctest, 1 test, 0 failures
+
+Randomized with seed 484163
+```
+
+You can see that we have `1 doctest` and `1 test` (1 "regular" test) with 0
+failures.
+
+We are not going to cover doctests in this example but if you would like to
+learn more about them then check out the following link,
+https://elixir-lang.org/getting-started/mix-otp/docs-tests-and-with.html#doctests.
+
+Now let's go and check out the code.
+
+In a text editor of your choice, open up the `lib/codecov_example.ex` (your
+name will vary if you didn't call your project the same thing as me but the file
+will match your project name).
+
+You should see something that looks similar to this...
+```elixir
+defmodule CodecovExample do
+  @moduledoc """
+  Documentation for CodecovExample.
+  """
+
+  @doc """
+  Hello world.
+
+  ## Examples
+
+      iex> CodecovExample.hello()
+      :world
+
+  """
+  def hello do
+    :world
+  end
+end
+```
+
+Let's delete all the code with the exception of the first and last lines.
+
+Now, let's add our own functions to this file. Copy and paste the following
+functions into your module...
+```elixir
+def add_one(n), do: 1 + n
+
+def add_two(n), do: 2 + n
+
+def add_three(n), do: 3 + n
+```
+
+You should now have something that looks like this...
+```elixir
+defmodule CodecovExample do
+
+  def add_one(n), do: 1 + n
+
+  def add_two(n), do: 2 + n
+
+  def add_three(n), do: 3 + n
+end
+```
+
+As you can see, these are fairly straightforward functions so I won't explain
+them too much. They all just do what their name says they do.
+
+Now let's go and add tests for these. Open `test/codecov_example_test.exs`.
+
+Delete the default test for the `hello/1` function (as we deleted this function
+earlier) and replace it with the following...
+
+```elixir
+  describe "testing the add functions in CodecovExample" do
+    test "adds one" do
+      assert CodecovExample.add_one(10) == 11
+    end
+
+    test "adds two" do
+      assert CodecovExample.add_two(10) == 12
+    end
+  end
+```
+
+As you can see, this has added functions that test our `add_one/1` and
```

Comment:
This readme will be linked in the main readme after the tests. (Again I forgot to add the link 🤦‍♀️. So whoever is reading it will have just read about tests)